src/selectors.ts

```diff
@@ -184,7 +185,7 @@ export class Selectors {
       if (eqIndex !== -1 && part.substring(0, eqIndex).trim().match(/^[a-zA-Z_0-9-+:]+$/)) {
         name = part.substring(0, eqIndex).trim();
         body = part.substring(eqIndex + 1);
-      } else if (part.startsWith('"')) {
+      } else if (hasTextSelectorSurroundings(part)) {
```

Comment:
Unfortunately, we do not have a way to share the code between injected (in the page) and selectors (in the node). I'd just copy it here.

Note that you changing semantics from 'starts with' to 'starts and ends with' - this should be reflected in the docs.